src/analysis/properties.rs

```diff
@@ -239,6 +240,9 @@ fn analyze_property(
         if let Ok(ref s) = used_rust_type(env, prop.typ, true) {
             imports.add_used_type(s);
         }
+        if type_string.is_ok() {
+            imports.add("glib::ToValue");
```

Comment:
Doesn't this also requires ToValueOptional?

Reply:
No, that's only used for a blanket `ToValue` implementation. As a user you'd always either use the `ToValue` trait or the `Value::get` (etc) API so no other trait needs to be in scope.

Reply:
Makes sense, thanks !